gnomad_parse/parse_gnomad_vcf.wdl

```diff
@@ -0,0 +1,128 @@
+version 1.0
+
+workflow parse_gnomad_vcf {
+
+    input {
+        Array[File] vcfs
+        String docker
+    }
+
+    scatter(vcf in vcfs) {
+        call parse {
+            input:
+                vcf = vcf,
+                docker = docker
+        }
+    }
+
+    call concat {
+        input:
+            parsed = parse.parsed,
+            docker = docker
+    }
+
+    output {
+        Array[File] parsed = parse.parsed
+        File merged = concat.merged
+    }
+}
+
+task parse {
+
+    input {
+        File vcf
+        String docker
+    }
+
+    String base = sub(sub(basename(vcf, ".bgz"), ".gz$", ""), ".vcf$", "")
+    Int local_disk = ceil(size(vcf, "G") * 2 + 2)
+
+    command <<<
+        
+        set -euxo pipefail
+
+        bcftools query -H -f '%CHROM\t%POS\t%REF\t%ALT\t%ID\t%INFO/AF_fin\t%INFO/AF_nfe\t%INFO/AF_afr\t%INFO/AF_amr\t%INFO/AF_asj\t%INFO/AF_eas\t%INFO/AF_mid\t%INFO/AF_sas\t%INFO/AC_fin\t%INFO/AC_nfe\t%INFO/AC_afr\t%INFO/AC_amr\t%INFO/AC_asj\t%INFO/AC_eas\t%INFO/AC_mid\t%INFO/AC_sas\t%INFO/AN_fin\t%INFO/AN_nfe\t%INFO/AN_afr\t%INFO/AN_amr\t%INFO/AN_asj\t%INFO/AN_eas\t%INFO/AN_mid\t%INFO/AN_sas\n' ~{vcf} | \
```

Comment:
Instead of hardcoding these could add the info tags as input param to make more modular.

Reply:
yea why nnot if you can be arsed... looks fine otherwise